### PR TITLE
remove redbaron and update minor dependencies

### PR DESCRIPTION
**Pull request by ryanpetrello**
redbaron is no longer needed by awx because it was only used by the `awx-manage migrate_to_database_settings` command (which we removed in https://github.com/ansible/awx/pull/3100/)

additionally, this PR bumps some minor AWX dependencies

### CODE REVIEW

File: requirements/requirements.in [hunk 1]
```diff
@@ -22,7 +22,7 @@ django-taggit==0.22.2
 djangorestframework==3.7.7
 djangorestframework-yaml==1.0.3
 irc==16.2
-jinja2=2.10
+jinja2==2.10
```
Comment (ryanpetrello) [inline requirements/requirements.in:25]: This is a bug introduced by me :'(

**Comment by ryanpetrello** [comment]
This PR is intended to replace/supercede https://github.com/ansible/awx/pull/3111

**Comment by ryanpetrello** [comment]
recheck

**Review by jakemcdermott — APPROVED** [review]

**Review by matburt — APPROVED** [review]
lgtm

**Pull request merged**